qa/tasks/vstart_runner.py

```diff
@@ -354,10 +354,19 @@ def hostname(self):
             self._hostname = 'localhost'
         return self._hostname
 
-    def get_file(self, path, sudo, dest_dir):
-        tmpfile = tempfile.NamedTemporaryFile(delete=False).name
-        shutil.copy(path, tmpfile)
-        return tmpfile
+    def get_file(self, path, sudo=False, dest_dir='/tmp'):
+        if dest_dir == '/tmp':
+            # If we're storing in /tmp, generate a unique filename
+            (_fd, local_path) = tempfile.mkstemp(dir=dest_dir)
+        else:
+            # If we are storing somewhere other than /tmp, use the original
+            # filename
+            local_path = os.path.join(dest_dir, path.split(os.path.sep)[-1])
+        try:
+            shutil.copy(path, local_path)
+        except shutil.SameFileError:
+            pass
+        return local_path
```

Comment:
Okay from my understanding, the change here basically adds a bunch of default values to the argument, then make use of sudo and dest_dir if the user specifies.

I think we can log.info("path and local_path refers to the same file") in the exception?